__tests__/app.test.js

```diff
@@ -220,4 +220,70 @@ describe('POST /api/articles/:article_id/comments', () => {
             expect(response.body.msg).toEqual('bad request');
         });
     });
-});
\ No newline at end of file
+});
+
+
+describe('PATCH /api/article/:article_id', () => {
+    test('responds with updated article object', () => {
+        const articleUpdate = { inc_votes: 150 };
+
+        return request(app)
+        .patch('/api/articles/1')
+        .send(articleUpdate)
+        .expect(200)
+        .then((response) => {
+            const responseArticle = response.body.article;
+            expect(responseArticle).toEqual({
+                article_id: 1,
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: "2020-07-09T20:11:00.000Z",
+                votes: 250,
+                article_img_url:
+                "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+            });
+        });
+    });
+    test('article has its vote count updated by given value', () => {
+        const articleUpdate = { inc_votes: 150 };
+
+        return request(app)
+        .patch('/api/articles/1')
+        .send(articleUpdate)
+        .expect(200)
+        .then(() => {
+            return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .then((response) => {
+                const article = response.body.article;
+
+                expect(article.votes).toBe(250);
+            });
+        });
+    });
+    test('returns 404 error message if given correctly formatted article ID that does not exist', () => {
+        const articleUpdate = { inc_votes: 150 };
+
+        return request(app)
+        .patch('/api/articles/1322334432')
+        .send(articleUpdate)
+        .expect(404)
+        .then((response) => {
+            expect(response.body.msg).toBe('not found');
+        });
+    });
+    test('returns 400 error message if given article ID has invalid format', () => {
+        const articleUpdate = { inc_votes: 150 };
+
+        return request(app)
+        .patch('/api/articles/AUUGGGHHH')
+        .send(articleUpdate)
+        .expect(400)
+        .then((response) => {
+            expect(response.body.msg).toBe('bad request');
+        });
+    });
```

Comment:
Some other tests to consider are:
Status 200, updates single article object (decrease votes)
Status 400, incorrect body, e.g. inc_votes property is not a number
(Optional) Status 200, if inc_votes is missing, should not update article and return original single article object (this may be included as a 400 but that could make for a brittle test suite)